src/siwe.ts

```diff
@@ -86,6 +88,24 @@ export class SiweMessage {
     }
   }
 
+  static fromCacao(cacao: Cacao): SiweMessage {
+    const account = AccountId.parse(cacao.p.iss.replace('did:pkh:', ''))
+    return new SiweMessage({
+      domain: cacao.p.domain,
+      address: account.address,
+      statement: cacao.p.statement,
+      uri: cacao.p.aud,
+      version: cacao.p.version,
+      nonce: cacao.p.nonce,
+      issuedAt: new Date(cacao.p.iat).toISOString(),
+      expirationTime: cacao.p.exp ? new Date(cacao.p.exp).toISOString() : undefined,
+      notBefore: cacao.p.nbf ? new Date(cacao.p.nbf).toISOString() : undefined,
```

Comment:
We should not need to use `Date` here. Since the string will already be in the correct format.